yarn-project/ethereum/src/eth_cheat_codes.ts

```diff
@@ -373,6 +373,7 @@ export class EthCheatCodes {
    * @param blockNumber - The block number that's going to be the new tip
    */
   public reorgTo(blockNumber: number): Promise<void> {
+    this.logger.info('reorgTo', { blockNumber });
```

Comment:
Based on depth being + 1, my guess would be that it is what might have caused your the pain above? 👀 


If I want the same block that I am to be the new tip, the depth would be +1, and end up rolling back the block so kinda feels like it.